backend/msgpack/server.go

```diff
@@ -48,11 +54,14 @@ const (
 type server struct {
 	sync.RWMutex
 
-	opts             ServerOptions
-	writerMgr        instanceWriterManager
-	shardFn          ShardFn
-	placementWatcher services.StagedPlacementWatcher
-	state            serverState
+	opts                       ServerOptions
+	nowFn                      clock.NowFn
+	shardCutoverWarmupDuration time.Duration
```

Comment:
My personal preference is to use `time.duration` for durations and `...Nanos` for time, since the `time.Duration` is a type alias for `int64` so is equally efficient but is more self explanatory, and using nanos for time is time zone agnostic and provides space savings.